euth_wagtail/settings/base.py

```diff
@@ -365,6 +365,10 @@
 
 WAGTAIL_SITE_NAME = "euth_wagtail"
 
+# Base URL to use when referring to full URLs within the Wagtail admin backend -
+# e.g. in notification emails. Don't include '/admin' or a trailing slash
+WAGTAILADMIN_BASE_URL = "http://localhost:8000"
+
```

Comment:
@goapunk I added this and wasn't sure if it also meant we would have to overwrite in our pillar but I assume not?

Reply:
Yes, we do! Will add that as a task.